### PR TITLE
Add get_data method to Evoked

### LINKED ISSUE #9244: ENH: Add evoked.get_data
In #9241 there are lines like:
```
evoked_eeg_interp.data[evoked_eeg_interp.ch_names.index('EEG 004'), :]
```
We should have `evoked.get_data` like we have for epochs and raw so this can just become:
```
evoked_eeg_interp.get_data('EEG 004')
```
and become more API-consistent.

### PR DESCRIPTION
**Pull request by sappelhoff**
fixes #9244 

working on #9553 I thought I'd give this a shot as well. Please let me know what I should add - I can add the `units` parameter here or in #9553, depending on which one gets merged earlier.

~EDIT: the tmin/tmax params from https://github.com/mne-tools/mne-python/issues/9548 could also be added here.~ better in #9556 

To Do
- [x] have somebody tell me what's missing, or if this is really all that was needed
- [x] add whatsnew

### CODE REVIEW

File: mne/evoked.py [hunk 1]
```diff
@@ -158,6 +158,23 @@ def data(self, data):
...
+        """Get evoked data as 2D array.
+
+        Parameters
+        ----------
+        %(picks_all)s
+
+        Returns
+        -------
+        data : ndarray, shape (n_channels, n_times)
+            A view on evoked data.
```
Comment (larsoner) [inline mne/evoked.py:172]:
```suggestion
            A view on evoked data.

        Notes
        -----
        .. versionadded:: 0.24
```

**Review by larsoner — CHANGES_REQUESTED** [review]
Needs `latest.inc` update and a `versionadded` would be good, otherwise LGTM!

File: mne/evoked.py (hunk 1 above)
Comment (sappelhoff) [reply mne/evoked.py:172]: I squeezed that into the whatsnew commit. thanks

**Review by larsoner — APPROVED** [review]

**Comment by larsoner** [comment]
pip pre failure is https://github.com/numpy/numpy/issues/19469 and we can ignore it for now

**Review by agramfort — APPROVED** [review]

**Comment by drammock** [comment]
thanks @sappelhoff

**Pull request merged**